cmd/develop/main.go

```diff
@@ -0,0 +1,20 @@
+
+package main
+
+import (
+	"log"
+
+	"github.com/snyk/cli-extension-sbom/pkg/sbom"
+	"github.com/snyk/go-application-framework/pkg/devtools"
```

Comment:
nit: grouping of imports into builtin, external, internal
```suggestion
	"github.com/snyk/go-application-framework/pkg/devtools"
	
	"github.com/snyk/cli-extension-sbom/pkg/sbom"
```